dask/dataframe/tests/test_shuffle.py

```diff
@@ -916,3 +916,47 @@ def test_set_index_timestamp():
 
     assert_eq(df2, ddf_new_div)
     assert_eq(df2, ddf.set_index("A"))
+
+
+@pytest.mark.parametrize("compression", [None, "ZLib"])
+def test_disk_shuffle_with_compression_option(compression):
+    # test if dataframe shuffle works both with and without compression
+    with dask.config.set({"dataframe.shuffle-compression": compression}):
+        test_shuffle("disk")
+
+
+@pytest.mark.parametrize("compression", ["UNKOWN_COMPRESSION_ALGO"])
+def test_disk_shuffle_with_unknown_compression(compression):
+    # test if dask raises an error in case of fault config string
+    with dask.config.set({"dataframe.shuffle-compression": compression}):
+        with pytest.raises(ImportError) as excinfo:
+            test_shuffle("disk")
+        assert str(excinfo.value) == (
+            "Not able to import and load {0} as compression algorithm."
+            "Please check if the library is installed and supported by Partd.".format(
+                compression
+            )
+        )
+
+
+def test_disk_shuffle_check_actual_compression():
+    # test if the compression switch is really respected by testing the size of the actual partd-data on disk
+    def generate_raw_partd_file(compression):
+        # generate and write a dummy dataframe to disk and return the raw data bytes
+        df1 = pd.DataFrame({"a": list(range(10000))})
+        df1["b"] = (df1["a"] * 123).astype(str)
+        with dask.config.set({"dataframe.shuffle-compression": compression}):
+            f = maybe_buffered_partd(buffer=False, tempdir=None)
```

Comment:
```suggestion
            p1 = maybe_buffered_partd(buffer=False, tempdir=None)()
```

Or do you resuse the temporary variable `f`?